ci/Dockerfile

```diff
@@ -7,16 +7,16 @@ RUN mkdir /home/hail-ci && \
     chown hail-ci:hail-ci /home/hail-ci
 
 WORKDIR /batch
-COPY batch/batch/batch batch
```

Comment:
wat

Reply:
probably best explained in person, so let's pick it up tomorrow.